build-tools/debian-metadata/copyright

```diff
@@ -0,0 +1,171 @@
+Xamarin.Android SDK
```

Comment:
Regarding future maintenance, would it instead be possible to make this a symlink to the top-level `LICENSE` file, or -- failing that -- use some Makefile voodoo so that `LICENSE` is copied to `build-tools/debian-metadata/copyright`?

I'm not looking forward to needing to maintain *two* separate copies of license information.